docs/HowTo/Monitor/Elastic-Stack.md

```diff
@@ -4,35 +4,31 @@ description: Using Elastick Stack (ELK) with Hyperledger Besu
 
 # Elastic Stack
 
-Elastic Stack (also known as ELK) is a common open-source log management platform.
+[Elastic Stack] (ELK) is an open-source log management platform that is available when using the
+[Developer Quickstart](../../Tutorials/Developer-Quickstart.md).
 
-The [Quorum Developer Quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) provides example
-implementations using Elastic Stack for log management.
+The [Filebeat] configuration ingests logs and the [Metricbeat] configuration collects metrics from the nodes at regular
+defined intervals and outputs them to Redis for storage.
+Redis provides a highly available mechanism enabling storage by any of the Elastic Beats and pulled by Logstash as required.
 
-The [Filebeat] configuration ingests logs and the [Metricbeat] configuration collects metrics from
-the nodes at regular defined intervals and outputs them to Redis for storage. Redis provides a
-highly available mechanism enabling storage by any of the Elastic Beats and pulled by Logstash
-as required.
-
-The [pipeline configuration] defines the JSON format used for Besu logs and automatically picks up
-any new log fields.
+The [pipeline configuration] defines the JSON format used for Besu logs and automatically picks up any new log fields.
 
 !!! note
 
-    The pipeline configuration must match the your log format. If using the default, you can use
-    the [grok plugin](https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html)
+    The pipeline configuration must match the your log format.
+    If using the default, you can use the [grok plugin](https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html)
```

Comment:
"If using the default, you can use..." Does this refer to the default log format? Maybe specify this.